src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,42 @@
-import React from 'react';
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import { Post } from '../../types';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+interface Props {
+  post: Post;
+}
+
+export const PostInfo = ({
+  post: {
+    title,
+    user,
+    userId,
+    body,
+    comments,
+  },
+} : Props) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {title}
+      </h3>
+
+      {user && <UserInfo key={userId} user={user} />}
```

Comment:
What's the purpose of `key` here? You are not mapping through any array

For Your Information: you can set `key` for non-list elements, but you have to do it consciously. By doing so, you can manually control when the component should re-render.